libutils/json-utils.c

```diff
@@ -0,0 +1,263 @@
+#include <json-utils.h>
+#include <logging.h>    // Log()
+#include <file_lib.h>   // safe_fopen()
+#include <string_lib.h> // TrimWhitespace()
+#include <csv_parser.h>
+#include <alloc.h>
+#define CF_BUFSIZE 4096
```

Comment:
don't redefine an existing define. Maybe create your own define, or just use the number you want if you use it once.